### DISCUSSION TITLE
Enabling cordova plugins with correct permissions and settings

### DISCUSSION THREAD

**Discussion by abierbaum** [ionic-team/capacitor#3529] [Q&A] [answered]
I am new to using Cordova plugins with capacitor.  I working on a new Ionic application using angular that needs to access BLE support.  One area that is confusing for me is how to configure the permissions needed by the BLE plugin [1].  I see from the documentation here [2] that some manual configuration is likely needed for the plugin.  But being completely new to both capacitor and cordova I don't really know where to start.  Is there any tutorial to help someone like me to ensure the normal plugin variables and settings get set manually?

1: https://github.com/randdusing/cordova-plugin-bluetoothle
2: https://capacitorjs.com/docs/cordova/using-cordova-plugins#important-configuration

**Comment by imhoffd** [comment]
There's [this tutorial](https://www.joshmorony.com/migrating-cordova-plugins-to-capacitor-android/) by Josh Morony, but the problem is each plugin is different, so you'll need to apply the tutorial to your use case.

**Comment by jcesarmobile** [comment, marked as answer]
If the Cordova plugin was already adding the permissions to the Cordova apps by using the `config-file` tag, they should be added automatically to the Capacitor app, and the bluetoothle plugin is doing it, so that shouldn't be a problem in this case
https://github.com/randdusing/cordova-plugin-bluetoothle/blob/master/plugin.xml#L23-L28

**Reply by abierbaum** [reply]
Thanks for that clarification.  I didn't understand that this was happening.
